powerline/segments/common/net.py

```diff
@@ -74,6 +74,7 @@ def internal_ip(pl, interface='auto', ipv=4):
 	_interface_starts = {
 		'eth':      10,  # Regular ethernet adapters         : eth1
 		'enp':      10,  # Regular ethernet adapters, Gentoo : enp2s0
+        'en':       10,  # OS X                              : en0 
```

Comment:
@kamushadenes I have checked and you are using spaces. You must change this to tabs. Alignment is OK, but indentation is wrong.